test/associative/bharshbarg/arrays/diffDomZip.chpl

```diff
@@ -29,7 +29,6 @@ while !different {
   for (aa, bb) in zip(A,B) do
     if aa != bb {
       different = true;
-      break;
```

Comment:
Is removing the break just a workaround, or is there something wrong with breaking from a loop like this?

Reply:
Yeah, it's just a workaround.